tests/providers/test_phone_number.py

```diff
@@ -68,20 +68,27 @@ def setUp(self):
         self.fake = Faker('pt_br')
         Faker.seed(0)
 
+    def test_phone_number(self):
+        phone_number = self.fake.phone_number()
+        re.match(r"^[+]?([0-9]{2})?[ ]?[(]?[0]?\d{2}[)]?[ ]?[09]?[ ]?\d{4}[ -]?\d{4}$", phone_number)
+
     def test_msisdn(self):
         msisdn = self.fake.msisdn()
-        formats = ('5511', '5521', '5531', '5541', '5551', '5561', '5571', '5581')
+        formats = ('5511', '5521', '5531', '5541', '5551', '5561', '5571', '5581', '5584')
 
         assert msisdn is not None
         assert isinstance(msisdn, str)
-        assert len(msisdn) == 13
+        assert len(msisdn) == 13 or len(msisdn) == 12
         assert msisdn.isdigit()
         assert msisdn[0:4] in formats
+        re.match(r"^[+]?([0-9]{2})?[ ]?[(]?[0]?\d{2}[)]?[ ]?[09]?[ ]?\d{4}[ -]?\d{4}$", msisdn)
+
```

Comment:
per pep8, we only keep one line between methods. Could you remove this empty line?

Reply:
Sure, didn't notice it.

It would be a good thing to set up pep8 verification on CI, with flake8 or some other lib

Reply:
We already have it in Travis